build.gradle

```diff
@@ -88,6 +90,15 @@ dependencies {
 	testCompile "com.willowtreeapps.assertk:assertk-jvm:$assertkVersion"
 	testCompile "org.jetbrains.kotlin:kotlin-reflect"
 	testCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+	testCompile "org.junit.jupiter:junit-jupiter-api:$junitJupiterVersion"
+	testRuntime "org.junit.jupiter:junit-jupiter-engine:$junitJupiterVersion"
+	testRuntime "org.junit.platform:junit-platform-launcher:$junitPlatformVersion"
```

Comment:
Hm. Does it really work as is?
I see in other projects we have a `useJUnitPlatform()` for test configuration.